src/inscribeit/create_inscription.cpp

```diff
@@ -331,13 +339,14 @@ void CreateInscriptionBuilder::RestoreTransactions()
     genesis_pubkeyscript << 1;
     genesis_pubkeyscript << m_destination_pk.value();
 
-    CMutableTransaction funding_tx;
+    CMutableTransaction funding_tx = CreateFundingTxTemplate();
     funding_tx.vin = {CTxIn(COutPoint(uint256S(*m_txid), *m_nout))};
     funding_tx.vin.front().scriptWitness.stack.push_back(m_utxo_sig.value());
+
     funding_tx.vout = {CTxOut(*m_amount, funding_pubkeyscript)};
```

Comment:
Do not replace the whole object and assign its fields instead

Reply:
Done